test/unit/constants.test.ts

```diff
@@ -0,0 +1,37 @@
+describe("BASE URL when", () => {
+	// Mock the process.env
+	beforeEach(() => {
+		jest.resetModules(); // Reset modules to clear cached values
+	});
+
+	describe("NODE_ENV = production", () => {
+		it("should return the production host URL", () => {
+			process.env.NODE_ENV = "production";
+
```

Comment:
very cool sir, using BASE_URL, it will be easy to maintain. I like how you organized the tests based on different NODE_ENV values. makes it easier to understand what each test is checking.